pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1925,6 +2056,37 @@ func buildHTTPConnectionManager(pluginParams *plugin.InputParams, httpOpts *http
 	return connectionManager
 }
 
+func buildThriftRatelimit(rlsCluster, domain string) *thrift_ratelimit.RateLimit {
+	thriftRateLimit := &thrift_ratelimit.RateLimit{
+		Domain:          domain,
+		Timeout:         ptypes.DurationProto(20 * time.Millisecond),
+		FailureModeDeny: false,
+		RateLimitService: &ratelimit.RateLimitServiceConfig{
+			GrpcService: &core.GrpcService{
+				TargetSpecifier: &core.GrpcService_EnvoyGrpc_{
+					EnvoyGrpc: &core.GrpcService_EnvoyGrpc{
+						ClusterName: rlsCluster,
+					},
+				},
+			},
+		},
+	}
+
+	if err := thriftRateLimit.Validate(); err != nil {
+		panic(err)
```

Comment:
I think we should remove this. We used to do this for other parts of pilot but realized crashing pilot, which may potentially bring down the whole cluster, for one bad config is probably a bad idea. Sending invalid config to Envoy will get rejected which is exposed in metrics and istioctl, so its pretty visible